ground/uavobjgenerator/uavobjectparser.cpp

```diff
@@ -404,6 +404,12 @@ QString UAVObjectParser::parseXML(QString& xml, QString& filename)
         // Calculate size
         calculateSize(info);
 
+        // Buffer on flight side is 256 bytes
+        // Additional overhead is 2+4+2+2+1+1=12
+        if(info->numBytes > 244)

```

Comment:
@mlyle would it be possible to replace this magic number with a #define or const?